build.sh

```diff
@@ -341,6 +341,9 @@ build_cross_arch_component()
         if [ "$__HostArch" == "x64" ]; then
             export CROSSCOMPILE=1
         fi
+    elif [[ ("$__BuildArch" == "arm64") && "$__CrossArch" == "x64" ]]; then
+         unset CROSSCOMPILE
```

Comment:
Why don't we `export CROSSCOMPILE=0` here?

As far as I understand, `CROSSCOMPILE==0` is meant to be for building crosscomponents for "the same-bitness architecture" as a developer machine. For example, we `export CROSSCOMPILE=1` for x86/arm build on Linux/x64 host and `export CROSSCOMPILE=0` for x86/arm build on Linux/x86 host

Reply:
If `CROSSCOMPILE` is set to anything the build will fail unless ROOTFS_DIR is set.  See #17349